### DISCUSSION TITLE
About Documentations

### DISCUSSION THREAD

**Discussion by emreakay** [rawilk/laravel-form-components#70] [General]
Hi,
I really liked your documentations.

How do you create docs from md files?

**Comment by rawilk** [comment]
I use a markdown renderer from Spatie. I have a job that stores the rendered markdown from each of my repos that have docs on my server that I can later fetch on show on the site.

You can see how it works here: https://github.com/rawilk/randallwilk.dev/blob/main/app/Docs/DocumentationContentParser.php
